packages/mobile/android/app/build.gradle

```diff
@@ -135,6 +135,7 @@ android {
         versionName "1.1.385"
         resValue "string", "build_config_package", "co.audius.app"
         resValue 'string', "CODE_PUSH_APK_BUILD_TIME", String.format("\"%d\"", System.currentTimeMillis())
+        resConfigs "en"
```

Comment:
solana-phone dev called out that the apk said it could support all locales, so this was the way to say we only support english. def a bit minor, but hey i guess helps?